0800-appendix-58067-gramophone.Rmd

```diff
@@ -98,9 +98,10 @@ assessment but had some study blocks that required a level II assessment
 PSCIS stream crossing `r as.character(my_site)` was ranked as a high
 priority for follow up by the Canadian Wildlife Federation and
 @irvine2018AnalysisPriority because of significant amounts of habitat
```

Comment:
hmm. not super true. Prob should say that I said it was high because to Gollner et al prioritized it for restoration (due to high habitat value- if thats what they said)